okhttp-hpacktests/src/test/java/okhttp3/internal/http2/HpackDecodeTestBase.java

```diff
@@ -41,7 +41,7 @@
     for (String interopTestName : interopTests) {
       List<Story> stories = HpackJsonUtil.readStories(interopTestName);
       if (stories.isEmpty()) {
-        fail("No stories for: " + interopTestName);
+        Collections.emptyList();
```

Comment:
what does this do?

Reply:
So options we have

1) Like maven - don't build this module
2) Build and fail if git submodule is missing.  This is master currently.
3) Use test assumptions to (visibly) skip tests if not available.  This failed when the test suite generation throws an Assume failure :(
4) Return 0 test cases, (silently) skips tests but compiles the code etc.  Still runs locally etc.

Above is an implementation of option 4.  Option 3 was my first attempt on the PR.

Reply:
I think Jesse is referring to the fact that you don't assign this to a variable. It's a bare statement with no effect.

Reply:
Too much Kotlin.  apologies for the shoddy diffs recently.

Reply:
Same result luckily. since it loops through an empty collection next.